contracts/src/protocol/OmniAVS.sol

```diff
@@ -1,106 +1,169 @@
 // SPDX-License-Identifier: GPL-3.0-only
-pragma solidity 0.8.12;
+pragma solidity =0.8.12;
+
+import { OwnableUpgradeable } from "@openzeppelin-upgrades/contracts/access/OwnableUpgradeable.sol";
 
 import { DelegationManager } from "eigenlayer-contracts/src/contracts/core/DelegationManager.sol";
-import { ISignatureUtils } from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
 import { IStrategy } from "eigenlayer-contracts/src/contracts/interfaces/IStrategy.sol";
 import { IDelegationManager } from "eigenlayer-contracts/src/contracts/interfaces/IDelegationManager.sol";
-
-import { OperatorStateRetriever } from "eigenlayer-middleware/src/OperatorStateRetriever.sol";
-import { ServiceManagerBase } from "eigenlayer-middleware/src/ServiceManagerBase.sol";
-import { RegistryCoordinator } from "eigenlayer-middleware/src/RegistryCoordinator.sol";
-import { IRegistryCoordinator } from "eigenlayer-middleware/src/interfaces/IRegistryCoordinator.sol";
-import { IStakeRegistry } from "eigenlayer-middleware/src/interfaces/IStakeRegistry.sol";
+import { ISignatureUtils } from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
+import { IServiceManager } from "eigenlayer-middleware/src/interfaces/IServiceManager.sol";
 
 import { OmniPredeploys } from "../libraries/OmniPredeploys.sol";
 import { IOmniPortal } from "../interfaces/IOmniPortal.sol";
 import { IOmniEthRestaking } from "../interfaces/IOmniEthRestaking.sol";
 import { IOmniAVS } from "../interfaces/IOmniAVS.sol";
 import { IOmniAVSAdmin } from "../interfaces/IOmniAVSAdmin.sol";
 
-/**
- * @title OmniAVS
- * @notice Omni AVS contract. It is responsible for syncing Omni AVS operator
- *         stake and delegations with the Omni chain.
- */
-contract OmniAVS is IOmniAVS, IOmniAVSAdmin, ServiceManagerBase, OperatorStateRetriever {
-    /// @dev AVS Quorum numbers, Omni only has one quorum
-    bytes public constant QUORUM_NUMBERS = hex"00";
+contract OmniAVS is IOmniAVS, IOmniAVSAdmin, IServiceManager, OwnableUpgradeable {
+    /// @notice Constant used as a divisor in calculating weights
+    uint256 public constant WEIGHTING_DIVISOR = 1e18;
+
+    /// @dev OmniPortal.xcall gas limit per each validator in syncWithOmni
+    uint256 internal constant _XCALL_GAS_LIMIT_PER_VALIDATOR = 10_000;
```

Comment:
- why underscore here?
- should we make these state variables, so they can be updated for the restaking contract?